### ISSUE TITLE
bug - nagstamon attempts to launch and display to a screen that isn't active (undocked laptop)

### ISSUE THREAD

**Issue by thomasjamesgoin** [HenriWahl/Nagstamon#83]
I have a laptop with 3 displays attached to a dock at work and I generally use nagstamon on the VGA port (screen 3) while at work.  However, while at home I tend to have Nagstamon in icon view.  Sadly, Nagstamon doesn't detect screens properly when launching and has no fallback behavior such as "iconified" or "systray" on screen 0 when the screen it tries to attach to isn't valid.  This should probably be addressed.
```
~$ /usr/bin/nagstamon
Fontconfig warning: "/etc/fonts/conf.d/65-droid-sans-fonts.conf", line 103: Having multiple values in <test> isn't supported and may not work as expected
Fontconfig warning: "/etc/fonts/conf.d/65-droid-sans-fonts.conf", line 138: Having multiple values in <test> isn't supported and may not work as expected
Traceback (most recent call last):
  File "/usr/bin/nagstamon", line 125, in <module>
    output = GUI.GUI(conf=conf, servers=servers, Resources=Resources, debug_queue=debug_queue, GUILock=GUILock)
  File "/usr/share/nagstamon/Nagstamon/GUI.py", line 166, in __init__
    self.popwin = Popwin(conf=self.conf, output=self)
  File "/usr/share/nagstamon/Nagstamon/GUI.py", line 1848, in __init__
    self.SwitchMode()
  File "/usr/share/nagstamon/Nagstamon/GUI.py", line 1888, in SwitchMode
    x0, y0, width, height = self.output.monitors[int(self.output.conf.fullscreen_display)]
KeyError: 3
```

**Comment by berndwaibel** [comment]
I may have the same, but I am using "fullscreen".
May be you see 
https://github.com/HenriWahl/Nagstamon/issues/129

**Comment by HenriWahl** [comment]
Fixed in 87325a5d51369b0bc701246a595e8258220dd73d. If screen is missing jump back to 0,0.
